frontend/pages/faq.tsx

```diff
@@ -70,6 +55,7 @@ const useStyles = makeStyles((theme) => {
         fontSize: 14,
         fontWeight: "normal",
       },
+      paddingTop: "70px",
```

Comment:
If you agree with my improvements, the paddingTop is redundant and can be removed
```suggestion

```